src/Tribe/Tickets.php

```diff
@@ -2822,10 +2822,10 @@ protected function update_capacity( $ticket, $data, $save_type ) {
 			}
 
 			// set the default capacity to that of the event, if set, or to unlimited
-			$default_capacity = Tribe__Utils__Array::get( $data, 'event_capacity', -1 );
+			$default_capacity = (int) Tribe__Utils__Array::get( $data, 'event_capacity', -1 );
 
 			// Fetch capacity field, if we don't have it use default (defined above)
-			$data['capacity'] = trim( Tribe__Utils__Array::get( $data, 'capacity', $default_capacity ) );
+			$data['capacity'] = (int) trim( Tribe__Utils__Array::get( $data, 'capacity', $default_capacity ) );
```

Comment:
Not needed for this fix but also the conditional below needs it to be set as a string

```suggestion
			$data['capacity'] = trim( Tribe__Utils__Array::get( $data, 'capacity', $default_capacity ) );
```